admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -765,7 +771,7 @@ def _makeIndex(self, database, table):
         self._log.info('Generating index %r.%r', self.indexDb, metaTable)
 
         # try to delete existing table first
-        self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
+        #self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
```

Comment:
And this is exactly a problem with the distributed (parallel) data loading. Each instance of the loader will always attempt to drop the table before attempting to load it with its own portion of the index. This was not a problem for the various integration tests which were relying on a single run of the loader tool.

Perhaps we should consider controlling this behavior through an option passed to the loader script and passing this flag into the code?